src/engine/image.cpp

```diff
@@ -1120,6 +1120,55 @@ namespace fheroes2
         }
     }
 
+    void ApplyVerticalGradient( fheroes2::Image & image, const uint8_t outsideColor, const uint8_t insideColor, const uint8_t borderWidth, const uint8_t borderColor )
```

Comment:
We are already in `fheroes2` namespace so there is no need to type it for `fheroes2::Image`:
```suggestion
    void ApplyVerticalGradient( Image & image, const uint8_t outsideColor, const uint8_t insideColor, const uint8_t borderWidth, const uint8_t borderColor )
```